pkg/pool-manager/pod_pool.go

```diff
@@ -152,6 +153,69 @@ func (p *PoolManager) allocatePodFromPool(pod *corev1.Pod) (string, error) {
 	return macAddr.String(), nil
 }
 
+func (p *PoolManager) initPodMap() error {
+	log.V(1).Info("start InitMaps to reserve existing mac addresses before allocation new ones")
+	pods, err := p.kubeClient.CoreV1().Pods("").List(metav1.ListOptions{})
```

Comment:
AFAIK this will iterate only over pods in the working namespace, not all namespaces

Reply:
I test it now and I get all the pod in all namespaces with the `Pods("")`

Reply:
Oh, then it's good! Thanks